src/View/SidePanel.java

```diff
@@ -0,0 +1,53 @@
+package View;
+
+import java.awt.FlowLayout;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+import javax.swing.ImageIcon;
+import javax.swing.JComponent;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JTabbedPane;
+
+public class SidePanel extends JPanel {
+	
+	private JLabel stats;
+	
+	private JTabbedPane players;
+	
+	private JComponent p1Panel;
```

Comment:
Is there a way to make this dynamically sized. or all implemented in a single array?